### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by watson**

### CODE REVIEW

**Comment by watson** [comment]
jenkins run the module tests for ALL

File: lib/instrumentation/modules/cassandra-driver.js [hunk 1]
```diff
@@ -13,14 +13,31 @@ module.exports = function (cassandra, agent, { version, enabled }) {
...
     shimmer.wrap(cassandra.Client.prototype, 'eachRow', wrapEachRow)
     shimmer.wrap(cassandra.Client.prototype, 'batch', wrapBatch)
   }
 
   return cassandra
 
+  function wrapAsyncConnect (original) {
+    return async function wrappedAsyncConnect () {
+      const span = agent.startSpan('Cassandra: Connect', 'db', 'cassandra', 'connect')
+      const ret = await original.apply(this, arguments)
```
Comment (vigneshshanmugam) [inline lib/instrumentation/modules/cassandra-driver.js]: just a question, don't we need to explicitly add try catch here and end span?

Comment (watson) [reply lib/instrumentation/modules/cassandra-driver.js]: Fixed

**Comment by watson** [comment]
jenkins run the module tests for ALL

**Comment by watson** [comment]
jenkins run the module tests for ALL

**Comment by watson** [comment]
jenkins run the module tests for ALL

**Comment by watson** [comment]
jenkins run the module tests for ALL

**Comment by watson** [comment]
jenkins run the module tests for ALL

**Comment by watson** [comment]
jenkins run the module tests for ALL

File: lib/instrumentation/patch-async.js [hunk 2]
```diff
@@ -430,51 +433,65 @@ module.exports = function (ins) {
...
           if (typeof cb === 'function') cb = ins.bindFunction(cb)
           return fn.call(this, a, b, c, d, e, cb)
         }
+        break
       default:
-        return fallback
+        wrapper = fallback
     }
+
+    if (promisify.custom in fn) {
```
Comment (vigneshshanmugam) [inline lib/instrumentation/patch-async.js]: Wow, TIL

**Pull request merged**